tamr_unify_client/models/project/attribute_mapping_collection.py

```diff
@@ -0,0 +1,75 @@
+from tamr_unify_client.models.base_collection import BaseCollection
+from tamr_unify_client.models.project.attribute_mapping import AttributeMapping
+
+
+class AttributeMappingCollection(BaseCollection):
+    """Collection of :class'tamr_unify_client.models.project.attribute_mapping.AttributeMapping'
+    :param client: Client for API call delegation.
+    :type client: :class:`~tamr_unify_client.Client`
+    :param resource_json: JSON data representing this resource
+    :param api_path: API path used to access this collection.
+    E.g. ``"projects/4/attributeMappings/19054-12"``
+    :type api_path: str
+    """
+
+    def by_relative_id(self, relative_id):
+        """Retrieve an attribute mapping by relative ID.
+       :param relative_id: The relative ID.
+       :type relative_id: str
+       :returns: The specified attribute mapping.
+       :rtype: :class:`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping`
+       """
+        return super().by_relative_id(AttributeMapping, relative_id)
+
+    def by_external_id(self, external_id):
+        """Retrieve an attribute mapping by external ID.
+        Since attributes do not have external IDs, this method is not supported and will
+        raise a :class:`NotImplementedError` .
+        :param external_id: The external ID.
+        :type external_id: str
+        :returns: The specified attribute mapping, if found.
+        :rtype: :class:`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping`
+        :raises KeyError: If no attribute mapping with the specified external_id is found
+        :raises LookupError: If multiple attribute mappings with the specified external_id are found
+        :raises NotImplementedError: AttributeMapping des not support external_id
+        """
+        raise NotImplementedError("AttributeMapping des not support external_id")
+
+    def by_resource_id(self, resource_id):
+        """Retrieve an attribute mapping by resource ID.
+        Since attributes do not have resource IDs, this method is not supported and will
+        raise a :class:`NotImplementedError` .
+        :param resource_id: The resource ID.
+        :type resource_id: str
+        :returns: The specified attribute mapping, if found.
+        :rtype: :class:`~tamr_unify_client.models.project.attribute_mapping.AttributeMapping`
+        :raises KeyError: If no attribute mapping with the specified resource_id is found
+        :raises LookupError: If multiple attribute mappings with the specified resource_id are found
+        :raises NotImplementedError: AttributeMapping des not support resource_id
+        """
+        raise NotImplementedError("AttributeMapping des not support resource_id")
```

Comment:
I'm pretty sure no resources have a `resourceId` property in their `JSON`--it's just a relative ID of `self.api_path + resource_id`